public/index.html

```diff
@@ -22,7 +22,10 @@
     <meta name="description" content="MyFriendBen, a Colorado public benefits and public assistance screening tool." />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
-    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css?Open+Sans:300,400,700&family=Roboto:300,400,500,700&family=Roboto+Slab:700&display=swap"
```

Comment:
@Mdemenko1 This changes the main font style throughout the app.

Reply:
I am not seeing the change in the fonts through the app. Could you paste a screenshot where you are seeing this?